R-Interface/R/symlinkTools.R

```diff
@@ -216,7 +211,7 @@ convertAbsoluteSymlinksToRelative <- function(modulesRoot, renvCache)
     {
       linkLoc <- symlinks[row, "linkLocation"]
       setwd(dirname(linkLoc))
-      #print(paste0("For link '", padToMax(symlinks[row, "linkLocation"], 1), "' will convert '", padToMax(symlinks[row, "originalTarget"], 2), "' to '", padToMax(symlinks[row, "linkTarget"], 3), "'"))
+      print(paste0("For link '", padToMax(symlinks[row, "linkLocation"], 1), "' will convert '", padToMax(symlinks[row, "originalTarget"], 2), "' to '", padToMax(symlinks[row, "linkTarget"], 3), "'"))
```

Comment:
This is just here purely for debugging, but I figured with all the problems on mac this couldnt hurt either ;)

Reply:
this entire file hurts my eyes... but if it works I'll just never look at it again :) It's probably quite tricky to test all the os specific differences.

Reply:
Yes mine too... I think it would have been cleaner if Id done it in C++...